proto/src/state.rs

```diff
@@ -0,0 +1,5 @@
+#![cfg(feature = "penumbra-storage")]
```

Comment:
Will this only gate the `read` mod behind the feature flag?

Reply:
I _think_ the shebang should make it crate wide but let me double check that

Reply:
Yeah I think you're right: https://doc.rust-lang.org/reference/attributes.html

Reply:
Since there are `cfg` directives in the outer module, however, I think it would be cleaner to move this to `lib.rs` and have it attach to `mod state;` there, so that all of the `cfg` directives are in one place.